### PR TITLE
minor updates to file to update when the override parameter and/or th…

### PR DESCRIPTION
**Pull request by patrickingraham**
…e configurationOverride parameter are used.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -49,7 +49,7 @@ Configuration Interface Summary
 The standard interface for CSCs is defined in :lse:`209` and is the official reference/requirements document.
 This section provides a very brief overview of the key topics related to configuration that are described in further detail in the sections below.
 
-Configuration-related interactions with CSCs happens via two main events (``configurationsAvailable`` and ``configurationsApplied``) and a single parameter as part of the start command (``configurationOverride``).
+Configuration-related interactions with CSCs happens via two main events (``configurationsAvailable`` and ``configurationsApplied``) and a single parameter as part of the ``start`` command (``configurationOverride``).
```
Comment (mareuter) [inline index.rst]: configurationsApplied --> configurationApplied

**Review by mareuter — COMMENTED** [review]
Great, GH won't let me delete this comment as it wasn't supposed to be one. Sigh.

File: index.rst [hunk 2]
```diff
@@ -306,7 +312,7 @@ This example assumes the component of interest is already in the ``STANDBY`` sta
 
     # ATAOS must be in STANDBY state for this to work. All other CSCs will
     # use their default configurations
-    await atcs.enable(configurationOverride={'ATAOS': 'summit_constant_hex.yaml'})
+    await atcs.enable(override={'ATAOS': 'summit_constant_hex.yaml'})
```
Comment (mareuter) [inline index.rst:316]: This code example is out of date. The ATAOS does not provide this as an override file.

File: index.rst [hunk 3]
```diff
@@ -348,7 +354,7 @@ If working with an individual CSC, which should be a special case, the ``salobj.
     await atdome.start_task()
 
     await salobj.set_summary_state(
-    atdome, salobj.State.ENABLED, configurationOverride="simple_algorithm.yaml"
+    atdome, salobj.State.ENABLED, override="simple_algorithm.yaml"
```
Comment (mareuter) [inline index.rst:358]: This code example is out of date. The ATDome does not provide this as an override file.

File: index.rst [hunk 4]
```diff
@@ -558,11 +564,11 @@ Therefore, in cases where on-the-fly configuration changes are required, the fol
 
         await salobj.set_summary_state(ataos, salobj.State.ENABLED)
 
-    If an override configuration was modified/added, then you must specify it using the ``configurationOverride`` keyword
+    If an override configuration was modified/added, then you must specify it using the ``override`` keyword
 
     .. code-block:: python
 
-        await salobj.set_summary_state(ataos, salobj.State.ENABLED, configurationOverride='summit_constant_hex')
+        await salobj.set_summary_state(ataos, salobj.State.ENABLED, override='summit_constant_hex')
```
Comment (mareuter) [inline index.rst]: This code example is out of date. The ATAOS does not provide this as an override.

**Review by mareuter — COMMENTED** [review]
The override usages either need to be corrected because they are out of date or some text put in that these are representative examples and do not necessarily reflect the actual ones offered by the CSC.

File: index.rst (hunk 1 above)
Comment (patrickingraham) [reply index.rst]: There were three of these instances. All fixed.

**Comment by patrickingraham** [comment]
Thanks Michael.
Good point about the code. Because we already have examples in the _static directory which are linked in the text, I've added some text in a couple places saying that the text shows representative examples. 
It is a bit unfortunate we can't have it be real-world examples but I think it's just too hard to maintain.

File: index.rst (hunk 2 above)
Comment (patrickingraham) [reply index.rst:316]: see comment below

File: index.rst (hunk 3 above)
Comment (patrickingraham) [reply index.rst:358]: see comment below

File: index.rst (hunk 4 above)
Comment (patrickingraham) [reply index.rst]: see comment below

Comment (patrickingraham) [reply index.rst]: Note that I also added the file extension as it is now required.

**Review by mareuter — APPROVED** [review]
Thanks for the updates. Looks good to me.

File: index.rst [hunk 5]
```diff
@@ -723,10 +731,10 @@ Rules Regarding Configuration Definitions and Usage
 
     - See this :download:`example _summit.yaml <_static/_summit.yaml>` for the ATSpectrograph CSC.
     - This file is the second configuration file to get loaded by the CSC and will override any previously declared values.
-    - Providing the ``_<site>.yaml`` file (or any file with a ``_`` prefix) to the ``configurationOverride`` parameter must result in rejecting the command.
+    - Providing the ``_<site>.yaml`` file (or any file with a ``_`` prefix) to the ``configurationOverride`` parameter in the ``start`` command must result in rejecting the command.
     - The combination of the ``_<site>.yaml`` and ``_init.yaml`` files **must fully populate all configuration parameters**.
 
-#.  The override configuration files, if specified using the `configurationOverride` parameter in the start command, is the third file loaded and will override the values set by the previously configuration files.
+#.  The override configuration files, if specified using the `configurationOverride` parameter in the ``start`` command, is the third file loaded and will override the values set by the previously configuration files.
```
Comment (tribeiro) [inline index.rst]: double "\`" on `configurationOverride`?

**Review by tribeiro — APPROVED** [review]

**Pull request merged**